net_sink.go

```diff
@@ -260,15 +260,23 @@ func (s *netSink) FlushGauge(name string, value uint64) {
 }
 
 func (s *netSink) FlushTimer(name string, value float64) {
-	// Since we mistakenly use floating point values to represent time
-	// durations this method is often passed an integer encoded as a
+	s.flushFloatOptimized(name, "|ms\n", value)
+}
+
+func (s *netSink) FlushAggregatedTimer(name string, value, sampleRate float64) {
+	suffix := fmt.Sprintf("|ms|@%.1f\n", sampleRate)
```

Comment:
For performance reasons it would be great to avoid `fmt.Sprintf` here if possible.